styles/styles.css

```diff
@@ -0,0 +1,70 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+body {
+  max-width: 1440px;
+}
+
+.container {
+  padding: 120px 0;
+  margin: 0 auto;
+}
+.advantages {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+.row {
+  display: flex;
+  gap: 16px;
+  text-align: center;
+  align-items: flex-end;
+  margin: 0 auto;
+}
+.col {
+  width: 298px;
+  height: 178px;
+}
+.advantages--row {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+}
+.advantages--row__icon {
+  margin-bottom: 48px;
+  display: flex;
```

Comment:
Do these blocks have items? Display flex specifies how the block interacts with children only.